<ui version="4.0" >
 <class>loginDialogClass</class>
 <widget class="QDialog" name="loginDialogClass" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Account</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>:/icons/icq.png</normaloff>:/icons/icq.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="accountLabel" >
       <property name="text" >
        <string>Account:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>4</number>
       </property>
       <item>
        <widget class="QComboBox" name="accountBox" >
         <property name="editable" >
          <bool>true</bool>
         </property>
         <property name="maxVisibleItems" >
          <number>10</number>
         </property>
         <property name="frame" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="deleteButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
         <property name="icon" >
          <iconset resource="../qutim.qrc" >
           <normaloff>:/icons/crystal_project/delete_user.png</normaloff>:/icons/crystal_project/delete_user.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="passwordLabel" >
       <property name="text" >
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEdit" >
       <property name="toolTip" >
        <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Maximum length of ICQ password is limited to 8 characters.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="maxLength" >
        <number>32</number>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="saveBox" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Save my password</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="connectBox" >
       <property name="text" >
        <string>Autoconnect</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="securBox" >
       <property name="text" >
        <string>Secure login</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="openBox" >
       <property name="text" >
        <string>Show on startup</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="signButton" >
       <property name="text" >
        <string>Sign in</string>
       </property>
       <property name="icon" >
        <iconset resource="../qutim.qrc" >
         <normaloff>:/icons/crystal_project/signin.png</normaloff>:/icons/crystal_project/signin.png</iconset>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>accountBox</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>saveBox</tabstop>
  <tabstop>connectBox</tabstop>
  <tabstop>securBox</tabstop>
  <tabstop>signButton</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources>
  <include location="../qutim.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>signButton</sender>
   <signal>clicked()</signal>
   <receiver>loginDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>93</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel" >
     <x>6</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
